test/common/router/config_impl_test.cc

```diff
@@ -978,7 +979,8 @@ TEST_F(RouteMatcherTest, TestRoutes) {
     EXPECT_EQ("www2", route->clusterName());
     EXPECT_EQ("www2", virtualHostName(route));
     route->finalizeRequestHeaders(headers, stream_info, true);
-    EXPECT_EQ("/VxVct/pVth/fVr/rVgVx1", headers.get_(Http::Headers::get().Path));
+    // TODO(samflattery): uncomment after https://github.com/google/re2/issues/272 resolved
```

Comment:
Instead of commenting out the EXPECT_EQ lines, could you guard them with a check if FUZZING_MODE is not defined?

Reply:
Oh yeah that makes a lot more sense, I'll change that